examples/compose/plot_digits_pipe.py

```diff
@@ -25,20 +25,20 @@
 from sklearn.pipeline import Pipeline
 from sklearn.model_selection import GridSearchCV
 
-
 # Define a pipeline to search for the best combination of PCA truncation
 # and classifier regularization.
 pca = PCA()
 # set the tolerance to a large value to make the example faster
-logistic = LogisticRegression(max_iter=10000, tol=0.1)
+logistic = LogisticRegression(max_iter=1000, tol=0.2)
 pipe = Pipeline(steps=[("pca", pca), ("logistic", logistic)])
```

Comment:
Actually, we make a 101 rookie mistake here. The data are not scaled. We should not modify anything apart of adding a `StandardScaler` as a preprocessing stage:

```suggestion
from sklearn.preprocessing import StandardScaler
scaler = StandardScaler()
pipe = Pipeline(steps=[("scaler", scaler), ("pca", pca), ("logistic", logistic)])
```

I have a x5 speed up just by scaling the data because the `LogisticRegression` is converging faster.

Reply:
I tried a few different versions, here are the timing results:
- Original:
```
Best parameter (CV score=0.920):
{'logistic__C': 0.046415888336127774, 'pca__n_components': 45}
real 7.33
user 58.78
sys 6.04
```

- Using Standard Scaler:
```
Best parameter (CV score=0.924):
{'logistic__C': 0.046415888336127774, 'pca__n_components': 60}
real 3.46
user 24.25
sys 3.13
```
The graph changes:
![scaled](https://user-images.githubusercontent.com/28287747/142931991-5ca9642f-a3c6-485c-a476-0754a0dcf08e.png)

- Using Subset + higher tolerance:
```
Best parameter (CV score=0.942):
{'logistic__C': 1.0, 'pca__n_components': 45}
real 2.74
user 15.70
sys 2.22
```
  Do you think we should proceed with the Standard Scalar?

Reply:
> Do you think we should proceed with the Standard Scalar?

Yes, when I tried it was working quite well. We might need to check the description if the number of hyperparameter changes.

MNT -> Maintenance

Reply:
I tend to use DOC on these PRs though, but no strong feelings.